### PR TITLE
Add contributing

### PR DESCRIPTION
**Pull request by ArturBa**

### CODE REVIEW

**Comment by ArturBa** [comment]
Do you mind adding hacktoberfest badge on this? ;)

**Comment by rpdelaney** [comment]
Thank you for this.

It would be nice if the one test actually worked. Maybe this will light a fire for me to fix that.

cc @trapd00r Japh I'd like your feedback on this, especially re: the code of conduct this adds, if you have the time

**Comment by trapd00r** [comment]
> cc @trapd00r Japh I'd like your feedback on this, especially re: the code of conduct this adds, if you have the time

Hm. Do we really need a CoC? Who wrote it up? Standard boiler?

**Comment by rpdelaney** [comment]
> Do we really need a CoC? 

We probably don't _need_ one. I'm not exactly _against_ it though. Unfortunately I wasn't as specific as I should have been in #109 about what I had in mind.

> Who wrote it up?

Good question. @ArturBa ?

**Comment by ArturBa** [comment]
I've used some of a CoC template and adjust it to a project

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to LS_COLORS
+
+## Pull Request Process
+
+1. Add file extension and its color to [`LS_COLORS`](./LS_COLORS)
```
Comment (rpdelaney) [inline CONTRIBUTING.md:9]:
```suggestion
1. Add the new file extension and its color to [`LS_COLORS`](./LS_COLORS). Try to find the most appropriate category for the file type and match the style to the others in the same category.
```

**Review by rpdelaney — CHANGES_REQUESTED** [review]
I appreciate this. A CONTRIBUTING file is the logical place for this stuff. I think we don't need a full code of conduct just yet. Please remove it and I will be happy to merge this in. Thanks!

**Comment by rpdelaney** [comment]
Thank you!

**Pull request merged**